src/main/webapp/WEB-INF/jpa-configuration.xml

```diff
@@ -50,13 +50,17 @@
         </property>
     </bean>
 
-    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
-        <property name="driverClassName" value="${org.osiam.auth-server.db.driver}"/>
-        <property name="url" value="${org.osiam.auth-server.db.url}"/>
-        <property name="username" value="${org.osiam.auth-server.db.username}"/>
-        <property name="password" value="${org.osiam.auth-server.db.password}"/>
+    <bean id="hikariConfig" class="com.zaxxer.hikari.HikariConfig">
+        <property name="poolName" value="springHikariCP" />
```

Comment:
[Description of the poolName](https://github.com/brettwooldridge/HikariCP#frequently-used):
> This property represents a user-defined name for the connection pool and appears mainly in logging and JMX management consoles to identify pools and pool configurations. Default: auto-generated